server/index.ts

```diff
@@ -12,10 +34,10 @@ app.get('/api/hello', (_request, response) => {
 app.use(express.static('dist'));
 
 // Handle client routing, return all requests to the app
-app.get('*', (_request, response) => {
-  response.sendFile(path.join(__dirname, '../dist/index.html'));
-});
+// app.get('*', (_request, response) => {
+//   response.sendFile(path.join(__dirname, '../dist/index.html'));
+// });
```

Comment:
Why comment out? It will be required for later usage if you have client side routing